webserver/webserver/src/main/java/io/helidon/webserver/netty/NettyWebServer.java

```diff
@@ -75,7 +75,16 @@
     private volatile boolean started;
     private final AtomicBoolean shutdownThreadGroupsInitiated = new AtomicBoolean(false);
 
-    NettyWebServer(ServerConfiguration config,
+    /**
+     * Creates a new instance.
+     *
+     * @param config a server configuration instance
+     * @param routing       a default routing instance
+     * @param namedRoutings the named routings of the configured additional server sockets. If there is no
+     *                      named routing for a given named additional server socket configuration, a default
+     *                      routing is used.
+     */
+    public NettyWebServer(ServerConfiguration config,
                    Routing routing,
```

Comment:
This constructor also should not be public. We do not want users to invoke NettyWebServer, we want them to use `WebServer` factory methods or a builder.